packages/elements/src/slider/index.ts

```diff
@@ -1006,6 +1007,18 @@ export class Slider extends ControlElement {
     return this.fromNumber + this.minRangeNumber;
   }
 
+  /**
+   * Validate number field from changed thumb
+   * @returns {void}
+   */
+  private validateNumberField(): void {
+    const name = this.changedThumb?.getAttribute('name') as SliderDataName;
+    const numberField = this[`${name}Input`];
+    if (numberField) {
```

Comment:
```suggestion
    if (this.isShowInputField) {
```
numberField appearance depends on `showInputField` prop. I suggest to check on the props instead.